OptimizelySDK/Optimizely.cs

```diff
@@ -337,7 +339,13 @@ public Variation GetVariation(string experimentKey, string userId, UserAttribute
         /// <returns>A boolean value that indicates if the set completed successfully.</returns>
         public bool SetForcedVariation(string experimentKey, string userId, string variationKey)
         {
-            return Config.SetForcedVariation(experimentKey, userId, variationKey);
+            var inputValues = new Dictionary<string, string>
```

Comment:
If we are using it for input values, please name variable name correctly, the actual purpose of the variable.